gateway/integration_test.go

```diff
@@ -21,11 +21,13 @@ import (
 	"github.com/stretchr/testify/require"
 	"golang.org/x/sync/errgroup"
 
+	kitHelper "github.com/rudderlabs/rudder-go-kit/testhelper"
```

Comment:
[minor] more idiomatic for package name to be lower case only
```suggestion
	kithelper "github.com/rudderlabs/rudder-go-kit/testhelper"
```